### PR TITLE
Command shortcuts and bash completion

### PR DESCRIPTION
**Pull request by JoeReid**
Adds shortcut aliases to many commands (volume -> vol etc).
Adds hidden `install-bash-completion` command to generate and install the bash completion.

### CODE REVIEW

File: cli/command/cli.go [hunk 1]
```diff
@@ -208,3 +208,18 @@ func getServerHost(hosts []string, tls bool) (host string, err error) {
...
 	return
 }
+
+// Standard alias definitions
+var (
+	CreateAliases  = []string{"c", "cr", "cre"}
+	InspectAliases = []string{"i", "in", "ins", "insp"}
+	ListAliases    = []string{"list", "l", "li"}
+	UpdateAliases  = []string{"u", "up", "upd"}
+	RemoveAliases  = []string{"remove", "delete", "del", "d", "r"}
```
Comment (sosodin) [inline cli/command/cli.go]: would rm be a better choice instead of r.

Comment (JoeReid) [reply cli/command/cli.go]: `rm` is currently the default, these aliases are in addition to it

Comment (JoeReid) [reply cli/command/cli.go]: I agree that shortcuts should be decided upon from a product perspective (@croomes or @oicheryl ). Changing the aliases should be trivial, so I just added one that seemed sane.

Comment (sosodin) [reply cli/command/cli.go]: ^ Cool. Awaiting Cheryl's feedback...

**Pull request merged**